src/features/organizations/components/ActivistPortlHeader/index.tsx

```diff
@@ -56,14 +78,27 @@ const ActivistPortalHeader: FC<Props> = ({
         >
           {topLeftComponent}
           {user && (
-            <NextLink href="/my">
-              <ZUIPersonAvatar
-                firstName={user.first_name}
-                id={user.id}
-                isUser
-                lastName={user.last_name}
+            <>
+              <Button
+                onClick={(event) =>
+                  setLogoutMenuAnchorEl(
+                    (event.target as HTMLElement).closest('button')
```

Comment:
What is the purpose of the `closest()`? I'm guessing it's to consistently select the same HTML element regardless of which of the children captures the click. I wonder if it would be possible to use `event.currentTarget` instead (which should always be the root element of the component)?